src/lib/client.ts

```diff
@@ -158,13 +154,14 @@ export class Client implements IClient {
   }
 
   /* FARMER INTEGRATION */
-  public async startFarming(path: string, plotSizeGB: number): Promise<boolean> {
+  public async startFarming(path: string, plotSizeGB: number): Promise<void> {
     // convert GB to Bytes
     const plotSize = Math.round(plotSizeGB * 1024 * 1024 * 1024)
     const { rewardAddress } = (await config.read());
     if (rewardAddress === "") {
       util.errorLogger("Tried to send empty reward address to backend!")
     }
-    return await tauri.invoke("farming", { path, rewardAddress, plotSize })
+
+    return tauri.invoke("farming", { path, rewardAddress, plotSize });
```

Comment:
for `startFarming` and `startNode` -> you changed the signature from `Promise<boolean>` to `Promise<void>`.
We discussed it before, and if it will work like that, I'm pro for this simplification. 
However, the line `return tauri.invoke("farming", { path, rewardAddress, plotSize });` is still returning `true`/`false` (same for node), because the backend functions' signatures are the same.

Reply:
`invoke` returns `Promise<unknown>`, but we don't really care about value here, only if it resolves or not

Reply:
we don't use value in the `store.ts`: `await client.startFarming(this.plotDir, this.plotSizeGB);`